test/test-cases/hello-world/sku.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  target: 'actual-output'
```

Comment:
I agree. Also I want to be able to test scenarios which have no `sku.config.js`.